### PR TITLE
Create rake task to upload dependent/spouse ssn data

### PR DESCRIPTION
**Pull request by embarnard**
Co-authored-by: Tahsina Islam <tislam@codeforamerica.org>

### CODE REVIEW

File: lib/tasks/data_task_spouse_dependent_ssns.rake [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    client_ids = []
+    in_data.each do |record|
+      record = record.to_ary
+      client_id = record[0][1]&.strip
+      client_ids << client_id.to_i
+    end
+
+    temp_file = Tempfile.open(%w[temp_file .csv], "tmp/")
+    CSV.open(Rails.root.join(temp_file.path), 'wb') do |csv|
+      csv << %w{client_id ssn type}
```
Comment (embarnard) [inline lib/tasks/data_task_spouse_dependent_ssns.rake:17]: switch ssn and type

File: lib/tasks/data_task_spouse_dependent_ssns.rake [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    blob = ActiveStorage::Blob.create_and_upload!(
+      io: File.open(Rails.root.join(temp_file.path), 'rb'),
+      filename: "spouse_dependents_results.csv",
+      content_type: "text/csv"
+    )
+
+    puts "*******FILE KEY: #{blob.key}"
+    puts blob.inspect
+    puts "***********blob URL:"
+    temp_file.unlink
```
Comment (embarnard) [inline lib/tasks/data_task_spouse_dependent_ssns.rake:40]: probably not needed

File: lib/tasks/data_task_spouse_dependent_ssns.rake [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      record = record.to_ary
+      client_id = record[0][1]&.strip
+      client_ids << client_id.to_i
+    end
+
+    temp_file = Tempfile.open(%w[temp_file .csv], "tmp/")
+    CSV.open(Rails.root.join(temp_file.path), 'wb') do |csv|
+      csv << %w{client_id ssn type}
+
+      client_ids.each do |id|
```
Comment (embarnard) [inline lib/tasks/data_task_spouse_dependent_ssns.rake:19]: start with `Client.where(id: client_ids).includes(:intake, :dependents)` and each over that, so we don't have to fetch them one-by-one

maybe do .find_in_batches

File: lib/tasks/data_task_spouse_dependent_ssns.rake (hunk 2 above)
Comment (embarnard) [reply lib/tasks/data_task_spouse_dependent_ssns.rake:40]: https://apidock.com/ruby/Tempfile seeing here it is good practice to explicitly close and unlink and not wait for garbage collection

**Review by tgrathwell — APPROVED** [review]

**Pull request merged**